socs/agents/wiregrid_actuator/agent.py

```diff
@@ -242,6 +242,17 @@ def _insert_eject(
             self.log.error(msg)
             return False, msg
 
+        # Additional small moving to ensure the limit switch ON
+        # No limit switch check during this moving
+        # Only this function reflects pos/neg of <distance>.
+        # Please use +/- distance for insertion/ejection, respectively.
+        ret = self.actuator.move(0.5 * (float)(1. if is_insert else -1.), speedrate=0.2)
```

Comment:
```suggestion
        ret = self.actuator.move(0.5 * (1. if is_insert else -1.), speedrate=0.2)
```
The `(float)()` syntax is unusual in Python. We don't really need to be casting `1.` or `-1.` to `float` here anyway, so I'd just drop it.